x/gov/client/utils/query.go

```diff
@@ -346,6 +356,50 @@ func QueryProposerByTxQuery(clientCtx client.Context, proposalID uint64) (Propos
 	return Proposer{}, fmt.Errorf("failed to find the proposer for proposalID %d", proposalID)
 }
 
+// QueryInitialDepositByTxQuery will query for a initial deposit of a governance proposal by
+// ID.
+func QueryInitialDepositByTxQuery(clientCtx client.Context, proposalID uint64) (types.Deposit, error) {
```

Comment:
This function should not be public, e.g.

```suggestion
func queryInitialDepositByTxQuery(clientCtx client.Context, proposalID uint64) (types.Deposit, error) {
```

Reply:
Done